MentalHospital.Tests/Tests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Xunit;
+using Moq;
+using AutoMapper;
+using Shouldly;
+using MentalHospital.BLL.Mapper;
+using MentalHospital.BLL.Models;
+using MentalHospital.BLL.Services;
+using MentalHospital.DAL.Interfaces;
+using MentalHospital.DAL.Entities;
+
+
+namespace MentalHospital.Tests
+{
+    public class Tests
+    {
+        private readonly IMapper mapper = new Mapper(new MapperConfiguration(
```

Comment:
Better to not use real instances of things like automapper or fluent validation and so on, we are testing method logic, not automapper (leave real instance for now, will talk about this later)